build/yamls/base/agent.yml

```diff
@@ -13,6 +13,10 @@ spec:
     type: RollingUpdate
   template:
     metadata:
+      annotations:
+        prometheus.io/port: "10443"
```

Comment:
Could you explain what these annotations are for?

Reply:
@jianjuns the annotations are used by Prometheus service discovery to locate metrics listeners within Kubernetes. Sorry. I thought that I've replied that...

Reply:
Got it. Thanks!